gutenberg.php

```diff
@@ -495,7 +495,7 @@ function gutenberg_replace_default_add_new_button() {
 			}
 
 			var url = button.href;
-			var newUrl = url.replace( /&?classic-editor/, '' );
+			var newUrl = url.replace( /&?\?classic-editor/, '' );
```

Comment:
This regular expression assumes it must match a question mark. We should do one of two things:

- If it really must match question mark, remove `&` since we're not expecting a sequence of `&?`
- Make question mark optional

Maybe: `/[&\?]?classic-editor/`

https://regex101.com/r/SP3owo/1